### DISCUSSION TITLE
Multiple toolpathes export

### DISCUSSION THREAD

**Discussion by xPhilippeRoux** [HALRobotics/Beta#94] [Q&A]
Hi,

I would need to be able to export several toolpathes within the same project and name them automatically, e.g. "ModuleA" & "ModuleB" if I have 2 objects to export.

I can't find a way to do that, is it possible ?

Cheers.

**Comment by sebandraos** [comment]
@XtreeE-PRO , I'm not entirely sure what you mean. As it stands, any given Controller in a Session can only solve 1 Procedure at a time and therefore only Export 1 at a time. Procedures retain their names when Exported so you could probably use something like a StreamFilter in Gh with Solve and Export active to swap Procedures quickly and automatically export them.

**Reply by xPhilippeRoux** [reply]
Thanks for the answer @sebandraos.

This point would require a much broader explanation of my needs, therefore it might be helpful to organize a videoconference any time soon. Would it be possible ?
